content/doc/developer/blueocean-plugin-development/index.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: developerchapter
+summary: How to develop plugins for Blue Ocean
+wip: true
+title: Extending Blue Ocean
+references:
+- url: https://github.com/jenkinsci/blueocean-plugin
+  title: Main Blue Ocean plugin repository
+- url: https://github.com/jenkinsci/jenkins-design-language
+  title: Design language repository with the styles and common components for Blue Ocean
+---
+
+The Blue Ocean UX plugin for Jenkins is designed to be extended by other plugins. 
+Like many other plugins, Blue Ocean has its own extension points. 
+
+Blue Ocean is a Single Page Application, with front end aspects contributed by many plugins, 
```

Comment:
"with front-end aspects contributed by many plugins" is a bit too vague to be helpful IMHO.

Perhaps  "Blue Ocean is a single-page application which has different aspects of the application contributed by purpose-specific plugins, such as the Dashboard for Blue Ocean or i18n for Blue Ocean plugins."